code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -179,3 +179,58 @@
 	icon = 'icons/obj/eldritch.dmi'
 	icon_state = "eldrich_flask"
 	list_reagents = list(/datum/reagent/eldritch = 50)
+
+/obj/item/clothing/mask/void_mask
+	name = "Mask Of Madness"
+	desc = "Mask created from the suffering of existance, you can look down it's eyes, and notice something gazing back at you."
+	icon_state = "mad_mask"
+	w_class = WEIGHT_CLASS_SMALL
+	flags_cover = MASKCOVERSEYES
+	resistance_flags = FLAMMABLE
+	flags_inv = HIDEFACE|HIDEFACIALHAIR
+	///Who is wearing this
+	var/mob/living/carbon/human/local_user
+
+/obj/item/clothing/mask/void_mask/equipped(mob/user, slot)
+	. = ..()
+	if(slot != ITEM_SLOT_MASK)
+		return
+	if(ishuman(user) && user.mind && slot == ITEM_SLOT_MASK)
+		local_user = user
+		START_PROCESSING(SSobj,src)
+
+		if(IS_HERETIC(user) || IS_HERETIC_MONSTER(user))
+			return
+		ADD_TRAIT(src, TRAIT_NODROP, CLOTHING_TRAIT)
```

Comment:
```suggestion
	if(ishuman(user))
		if(user.mind && slot == ITEM_SLOT_MASK)
			local_user = user
			START_PROCESSING(SSobj,src)

			if(IS_HERETIC(user) || IS_HERETIC_MONSTER(user))
				return
			ADD_TRAIT(src, TRAIT_NODROP, CLOTHING_TRAIT)
```
if it doesn't have mind it's probably not human

Reply:
should it not stop checking user mind if ishuman(user) fails?

Reply:
Apparently it doesn't matter DM skips rest of checks anyway